modules/GenericPowermeter/manifest.yaml

```diff
@@ -23,6 +23,10 @@ provides:
 requires:
   serial_comm_hub:
     interface: serial_communication_hub
+  specific_power_meter:
```

Comment:
This needs a comment. It is not obvious why this is useful to a first time reader. Maybe the `implementation_id` could also be more specific. Something like "start_transaction_meter"? I do not love this either, but I have no better idea at the moment.

Reply:
friendly ping on this.